aait/backend/group-2B/Application/Features/Posts/Requests/Queries/GetPostsByTagsRequest.cs

```diff
@@ -0,0 +1,14 @@
+using MediatR;
+using SocialSync.Application.DTOs.PostDtos;
+
+namespace SocialSync.Application.Features.Posts.Requests.Queries;
+
+public class GetPostsByTagsRequest : IRequest<List<GeneralPostDto>>
```

Comment:
It would be better if we make Tags a property,
making Tags a property aligns with common coding conventions and design patterns. It provides a consistent approach for accessing and manipulating the identifier across different classes and codebases.

Possible change:

`public class GetPostsByTagsRequest : IRequest<List<GeneralPostDto>>
{
    public List<string> Tags { get; set; } = null!;
}
`

Reply:
Fixed.